core/types/block.libevm.go

```diff
@@ -45,7 +44,7 @@ func (h *Header) hooks() HeaderHooks {
 	return new(NOOPHeaderHooks)
 }
 
-func (e ExtraPayloads[HPtr, SA]) hooksFromHeader(h *Header) HeaderHooks {
+func (e ExtraPayloads[HPtr, BodyExtraPtr, SA]) hooksFromHeader(h *Header) HeaderHooks {
```

Comment:
@ARR4N can this be changed from `BodyExtraPtr` -> `BPtr` if we manage to merge the body and block extras together? This was as `BodyExtraPtr` in order to allow for a `BlockExtraPtr` without conflicting on the `B` letter.

Reply:
Yup, they'll be the same in my upcoming PR